package/koreader/package

```diff
@@ -2,15 +2,15 @@
 pkgname=koreader
 pkgdesc="KOReader"
```

Comment:
We could make this description more informative. Taking from koreader’s GitHub repo description, it could be something like _“Ebook reader application supporting PDF, DjVu, EPUB, FB2 and many more formats”_.

Reply:
We could use the one that the KOReader repository has:

> An ebook reader application supporting PDF, DjVu, EPUB, FB2 and many more formats.